electrum/commands.py

```diff
@@ -1322,6 +1322,34 @@ async def reverse_swap(self, lightning_amount, onchain_amount, wallet: Abstract_
             'onchain_amount': format_satoshis(onchain_amount_sat),
         }
 
+    @command('n')
+    async def convert_currency(self, from_amount, from_ccy = None, to_ccy = 'BTC'):
+        """Converts the given amount of currency to another using the
+        configured exchange rate source.
+        """
+        # Currency codes are uppercase
+        from_ccy = self.daemon.fx.ccy if from_ccy is None else from_ccy.upper()
+        to_ccy = to_ccy.upper()
+        # Get current rates
+        rate_from = self.daemon.fx.exchange.get_cached_spot_quote(from_ccy)
+        rate_to = self.daemon.fx.exchange.get_cached_spot_quote(to_ccy)
+        # Test if currencies exist
+        if rate_from.is_nan():
+              raise Exception('Currency to convert from is unknown')
+        if rate_to.is_nan():
+              raise Exception('Currency to convert to is unknown')
+        # Conversion
+        try:
+            amount = Decimal(from_amount) / rate_from * rate_to
+        except InvalidOperation:
+            raise Exception("from_amount is not a number")
+        return {
+            "amount": self.daemon.fx.ccy_amount_str(amount, False, to_ccy),
```

Comment:
I think we could return both `(from_amount, from_ccy)`, `(to_amount, to_ccy)` pairs.
```suggestion
            "from_amount": self.daemon.fx.ccy_amount_str(from_amount, False, from_ccy),
            "to_amount": self.daemon.fx.ccy_amount_str(amount, False, to_ccy),
```
(potentially even a timestamp to show how fresh the quote is, e.g. expose `fx.exchange._quotes_timestamp`, although it's not that important I guess)